lib/AST/Type.cpp

```diff
@@ -4281,6 +4281,14 @@ bool AnyFunctionType::hasSameExtInfoAs(const AnyFunctionType *otherFn) {
   return getExtInfo().isEqualTo(otherFn->getExtInfo(), useClangTypes(this));
 }
 
+bool swift::hasIsolatedParameter(ArrayRef<AnyFunctionType::Param> params) {
+  for (auto &param : params) {
```

Comment:
Can be simplified down to:

```
return llvm::any_of(params, [](auto &param) { return param.isIsolated(); });
```

Reply:
I'm going to ignore this one; it's just a style, not a real simplification.